packages/shorebird_cli/diff

```diff
@@ -0,0 +1 @@
+diff
```

Comment:
Did you mean to commit this?

Reply:
definitelly not, weird, I was sure I had removed, thanks for catching it, gonna remove and merge it